### ISSUE TITLE
Have Xcode highlight Podfile syntax in generated Pods.xcodeproj

### ISSUE THREAD

**Issue by nolanw** [CocoaPods/CocoaPods#4962]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md). 🌈

In Xcode, you can click on the `Podfile` in the `Pods` project, go to the File Inspector, and set "Type" to "Ruby Script". Now you have syntax highlighting! (After you close the file then reopen it. *sigh*)

It occurs to me that the generated `Pods.xcodeproj` could mark the `Podfile` with `explicitFileType = text.script.ruby`, then you get some syntax highlighting when editing the `Podfile` in Xcode. This would be quite handy.

Example from my `project.pbxproj` that successfully highlights syntax:
```
BABABABABABABABABABABABABABABABA /* Podfile */ = {isa = PBXFileReference;
  explicitFileType = text.script.ruby; includeInIndex = 1; name = Podfile;
  path = ../Podfile; sourceTree = SOURCE_ROOT;
  xcLanguageSpecificationIdentifier = xcode.lang.ruby; };
```

**Comment by segiddins** [comment]
👍🏻 a PR doing just that would be ace

**Comment by orta** [comment]
Yeah, this is an awesome idea

**Comment by segiddins** [comment]
Right now we set the `xc_language_specification_identifier` to `xcode.lang.ruby`, is that insufficient?

**Comment by nolanw** [comment]
Maybe that is sufficient! I opened this issue because I wasn't seeing `Podfile` syntax highlighting in an existing project. But I've tried making brand new projects here a couple times and the syntax highlighting for `Podfile` works fine without my doing anything.

I'll leave this issue open in case it's still worth adding the explicit file type. I'm not sure it's necessary. Not sure it hurts anything either?
